typescript/sdk/src/utils/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './chains';
```

Comment:
The one exception to my comment above about killing the index files. I kept this one cause there's so much in utils